tests/test_revision_ranges.py

```diff
@@ -0,0 +1,54 @@
+"""Test the revision_ranges module."""
+import unittest
+from pathlib import Path
+from unittest import mock
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Imports from package unittest are not grouped](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5481069)